### DISCUSSION TITLE
【求助】为什么我发的帖不见了

### DISCUSSION THREAD

**Discussion by wushanef** [share121/inter-knot#271] [其他]
我中午发了个帖，但是现在不见了，求大佬解答😭😭😭
![IMG_20240802_160529](https://github.com/user-attachments/assets/90673431-a7a3-41ba-961f-ff44e141ee62)

**Comment by wushanef** [comment]
这个委托报酬为1600菲林

**Comment by xxtg666** [comment]
<https://greasyfork.org/zh-CN/scripts/502874-%E7%BB%B3%E7%BD%91%E8%B7%A8%E5%9F%9F%E5%8A%A9%E6%89%8B/code>
根据第 43 行代码 `...discussions(first: 100)...` 可知只显示最新的 100 条帖子

**Reply by wushanef** [reply]
感谢大佬

**Comment by Mu-7593** [comment]
在github里找到帖子后给自己回复条消息试试能不能拉上来

**Reply by wushanef** [reply]
我试试

**Reply by wushanef** [reply]
谢谢大佬，成了
